### ISSUE TITLE
add helper function to replicate argument as a list

### ISSUE THREAD

**Issue by mcol** [R-Lum/Luminescence#299]
We often have code like this when a function self-calls:

```R
if(is(dose_points, "list")){
  dose_points <- rep(dose_points, length = length(object))
}else{
  dose_points <- rep(list(dose_points), length = length(object))
}
```
We could have a simple helper function that allows to write the following:
```R
dose_points <- .listify(dose_points, length = length(object))
```
implemented along these lines:
```R
.listify <- function(variable, length) {
  if (!is.list(variable))
    variable <- list(variable)
  rep(variable, length = length)
}
```

**Comment by RLumSK** [comment]
I had written that purpose the `Luminescence:::.expand_parameters()` function; if I am not mistaken it could also work here. I had not implemented this function after I had written it everywhere.

**Comment by mcol** [comment]
Thanks, I had not seen it. Your function is quite complicated, as it infers the parameters from the environment of the calling function. I've already seen a number of exceptions, where just some arguments are expanded but not others. 

For example, the `verbose` option is generally not expanded. And there are cases like this:
```R
if(is(list(...)$main, "list")){
  temp_main <- rep(list(...)$main, length = length(object))
}else{
  temp_main <- rep(list(list(...)$main), length = length(object))
}
```
I think being explicit about what we expand and having a really trivial implementation is quite an advantage (also seeing the problems we had when playing with `sys.calls()` and friends).

**Comment by RLumSK** [comment]
Yes, I know but it does not make much sense, as an example, to expand verbose. Besides, the expand function has a very limited application range. But if you feel that a more explicit approach is better, then please go ahead.
